dask_ml/model_selection/_incremental.py

```diff
@@ -817,6 +818,24 @@ class IncrementalSearchCV(BaseIncrementalSearchCV):
     >>> search = IncrementalSearchCV(model, params, random_state=0,
     ...                              n_initial_parameters=1000,
     ...                              patience=20, max_iter=100)
+
+    ``tol`` and ``patience`` are for stopping training on a "plateau", or when
```

Comment:
Is the term `plateau` likely to be well understood by novice users?  If yes then we should remove the quotes.  If no then we might want to consider avoiding the term and instead just describe it as you do after using it.  I might even start off this paragraph with 

"Often when training we get to a situation where additional work leads to little to no gain.  In these cases the ``tol`` and ``patience`` parameters can help us stop work early.  ..."